src/server/models/Volunteer.ts

```diff
@@ -0,0 +1,66 @@
+import {
+    Volunteer,
+  } from '@/types/volunteer';
+  import { Model, Schema, model, models } from 'mongoose';
+  
+  const VolunteerSchema = new Schema(
+    {
+      firstName: {
+        type: String,
+        required: true,
+      },
+      lastName: {
+        type: String,
+        required: true,
+      },
+      userName: {
+        type: String,
+        required: true,
+      },
+      id: {
+        type: Number,
+        required: true,
+      },
+      email: {
+        type: String,
+        required: true,
+      },
+      phoneNumber: {
+        type: Number,
+        required: true,
+      },
+      events: {
+        type: Schema.Types.ObjectId,
+        ref: 'Events', // link to the Events Schmea if made
+        required: false,
+      },
+      active: {
+        type: Boolean,
+        required: false,
+      },
+      hoursLogged: {
+        type: Schema.Types.ObjectId,
+        ref: 'Hours', // link to the Hours Schmea if made
+        required: false,
+      },
+      demographic: {
+        type: Schema.Types.ObjectId,
+        ref: 'Demographic', // link to the Demographic Schmea if made
+        required: true,
+      },
+      admin: {
+        type: Boolean,
+        required: true,
+      },
+    },
+    {
+      versionKey: false,
+      timestamps: true,
+    }
+  );
+  
+  export type VolunteerDocument = Volunteer & Document;
```

Comment:
Honestly this may be right, but I don't know what it does and it's not exactly the same in compassion ministries. Why did you do this differently?

Reply:
This is based off the first commit on compassion ministries.
This is the [link](https://github.com/hack4impact-utk/compassion-ministries/commit/2361ba247a2b2c9250739e000e7b7f79cac0683a).

Reply:
But yeah, I can update it to how it looks for them now. I don't think we need a VolunteerDocument.

Reply:
Well, actually they change it to a z.object, I don't know if thats the updated version, but I'll look into it.

Reply:
The line may not even be necessary. I will check with Andrew. You can leave it how you had it for now.

Reply:
I was confused, ignore the previous comments if you saw them (now deleted)